Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -40,6 +40,7 @@ extension XCScheme {
         public var selectedLauncherIdentifier: String
         public var buildConfiguration: String
         public var launchStyle: Style
+        public var askForAppToLaunch: Bool
```

Comment:
Since Xcode can open projects when `askForAppToLaunch` is not defined, I'd define it as optional here. That way if someone opens a project with XcodeProj where `askForAppToLaunch` is not defined, it won't get a diff after saving it.

Reply:
Also don't forget to include it in the [isEqual](https://github.com/tuist/XcodeProj/blob/6acfc0bc57fd86245025cfcf1f5e06d59b954ac5/Sources/XcodeProj/Scheme/XCScheme%2BLaunchAction.swift#L306) methods.

Reply:
Those comments apply for all the changes in this PR.

Reply:
Thank you for reviewing. Fixed them!